.banzaicloud/charts/spotguide-zeppelin/charts/zeppelin-spark/README.md

```diff
@@ -0,0 +1,83 @@
+# Zeppelin-Spark Chart
+
+This is a composite chart deploying the following sub-charts:
+
+- [Zeppelin](https://github.com/banzaicloud/banzai-charts/tree/master/zeppelin)
```

Comment:
Please update link to point the local chart

Reply:
will remove README files, as these are not standalone charts, they make sense only within the context of the spotguide.